pages/game.tsx

```diff
@@ -255,21 +254,23 @@ function Game() {
                     return "⬛️";
                   }
                 })}
-              </h1>
-            </div>
+              </Col>
+            </Row>
           );
         })}
-        {/* fill with gray squares for unused rows */}
-      </div>
+      </Container>
     );
 
     return (
-      <div>
-        <div className="text-centered">
-          You have already played today!
-          <div> Your daily Wordify board!{resultBoard}</div>
-        </div>
-      </div>
+      <Container className="mt-5 d-grid gap-2 text-center">
+        <Row>
+          <h1>You have already played today</h1>
```

Comment:
Should we consider rewording this to "Today's Wordify puzzle has been played" or something that would make more sense to be displayed right away after game over? Also - we can consider adding a copy to clipboard button that copies the "dashboard" to the user clipboard